core/utils/poly-look/src/react/chips/chips.jsx

```diff
@@ -1,21 +1,131 @@
-import React from "react";
+import React, { useState } from "react";
 
 import "./chips.css";
 
-const Chips = ({ chipsContent, activeChips, onChipClick, theme }) => {
+const chipsId = (e) => e.id || e.title || e;
+
+const othersId = "others";
+const allId = "all";
+
+/**
+ *
+ * Chips group
+ * @param {string[] || Object[]} chipsContent -  Content of the Chips
+ * @param {Object} [chipsContent.id] - Id of the chip (if chipsContent is an array of strings they act as id)
+ * @param {Object} [chipsContent.title] - Alternative for id of the chip (if chipsContent is an array of strings they act as id)
+ * @param {Object} [chipsContent.translation] - Translation of the chip (if chipsContent is an array of strings they act as translation)
+ * @param {string[]} defaultActiveChips - Chips that are active on load
+ * @param {callback} onChipClick - Chips onClick function (id of clicked chip, all active chips ids)
```

Comment:
If I'm understanding the logic of this correctly, this callback is going to do two things:
* Trigger whatever action when a chip is clicked
* Something with all active chips? Not clear what. 

At any rate, it seems to be a case of mixing two different functions in a single callback. The origin of this is that there's a single callback for chips with different logic, and you're putting all the logic here... Have you considered creating exclusive allChips and toggleChips (otherChips) logic and encapsulate it?